homework/index.js

```diff
@@ -1,6 +1,29 @@
 'use strict';
 
 {
+  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+
+  function createAndAppend(name, parent, options = {}) {
+    const elem = document.createElement(name);
+    parent.appendChild(elem);
+    Object.keys(options).forEach(key => {
+      const value = options[key];
+      if (key === 'text') {
+        elem.textContent = value;
+      } else if (key === 'html') {
+        elem.innerHTML = value;
+      } else {
+        elem.setAttribute(key, value);
+      }
+    });
+    return elem;
+  }
+  // ********************************************************
```

Comment:
Most developers, including myself,  would prefer a simple blank line between function definitions. Just like you would use a blank line, rather than a drawn line to separate paragraphs in a piece of text.